.github/workflows/scanner-e2e-test.yaml

```diff
@@ -0,0 +1,69 @@
+name: Scanner standalone E2E tests
+
+on:
+  workflow_call:
+
+jobs:
+  e2e-run:
+    name: "Deploy and run"
+    runs-on: ubuntu-latest
+    env:
+      SCANNER_E2E_QUAY_USERNAME: ${{ secrets.QUAY_RHACS_ENG_RO_USERNAME }}
+      SCANNER_E2E_QUAY_PASSWORD: ${{ secrets.QUAY_RHACS_ENG_RO_PASSWORD }}
+      SCANNER_E2E_GOOGLE_SA_TOKEN: ${{ secrets.GOOGLE_SA_CIRCLECI_SCANNER }}
+      # TODO Add registry.redhat.io secrets.
```

Comment:
what are the secrets below this then?

Reply:
AFAIK, they don't exist. I get empty strings with those references.

Reply:
`RH_REGISTRY_PASSWORD_RO` and `RH_REGISTRY_USERNAME_RO` can be used to access images from registry.redhat.io. We use them in https://github.com/stackrox/stackrox/blob/master/.github/workflows/verify-release.yml#L79-L84.